sherlockml/clients/base.py

```diff
@@ -108,3 +108,11 @@ def _get(self, endpoint, schema, **kwargs):
         response = self._get_raw(endpoint, **kwargs)
         _check_status(response)
         return _deserialise_response(schema, response)
+
+    def _post_raw(self, endpoint, *args, **kwargs):
```

Comment:
Since this function has a one-line body and is only being used once below in `_post`, can't we just incorporate its body into `_post`?

Reply:
We could, although the code implemented here is consistent with how the `get` methods are currently implemented.